analytical_env_cluster_config/steps.yaml.tpl

```diff
@@ -25,3 +25,9 @@ Steps:
     - "s3://${config_bucket}/scripts/emr/livy_client_conf.sh"
     Jar: "s3://eu-west-2.elasticmapreduce/libs/script-runner/script-runner.jar"
   ActionOnFailure: "CONTINUE"
+- Name: "get-scripts"
+  HadoopJarStep:
+    Args:
+    - "s3://${config_bucket}/scripts/emr/get_scripts.sh component/uc_repos /opt/emr"
```

Comment:
I'm a bit concerned about the destination deirectory, there are already files in /opt/emr, so to reduce the possibility of collisions, can we have a dest directory of /opt/emr/external.  ?  Then the get_scripts.sh could do a `mkdir -p` on the destination directory prior to copying.